### DISCUSSION TITLE
full width layout problem

### DISCUSSION THREAD

**Discussion by Chekki** [affiliatetheme/affiliatetheme#1] [General]
Hi,

after updating to 2.0 the header layout is boxed instead of full width: 

https://fernstudium-in-deutschland.de/

I would like to have a full width header. The setting show full width:

![Bildschirm­foto 2023-01-10 um 10 46 56](https://user-images.githubusercontent.com/23417760/211524507-0d13107a-9471-431a-930e-6a3afedb4616.jpg)
![Bildschirm­foto 2023-01-10 um 10 47 01](https://user-images.githubusercontent.com/23417760/211524515-39738c21-77f5-4e43-9b17-bece495a3b58.jpg)

Before the update:

![Bildschirm­foto 2023-01-10 um 11 20 02](https://user-images.githubusercontent.com/23417760/211524989-7ca4189a-95cb-4fca-9c2f-6fcbf04cc5f1.jpg)

After update:

![Bildschirm­foto 2023-01-10 um 10 47 24](https://user-images.githubusercontent.com/23417760/211525056-f89ce6d8-a1b8-491a-b15e-da08c7ad1537.jpg)

What can I do to get full width back?

Best regards

**Comment by endcoreAK** [comment]
@Chekki for bug reports please use the Github Issues feature 

See: [https://github.com/affiliatetheme/affiliatetheme/issues/2](https://github.com/affiliatetheme/affiliatetheme/issues/2)
